### PR TITLE
DOC-842: RC Reorg - Rework Admin menu, part II

### PR DESCRIPTION
**Pull request by lanceleonard**
- Initial draft

### CODE REVIEW

**Comment by lanceleonard** [comment]
A staged preview [is available](https://docs.redis.com/staging/jira-doc-842-ii/rc/databases/).

File: content/rc/databases/system-logs.md [hunk 1]
```diff
@@ -1,10 +1,12 @@
...
 Title: System logs
 description:
-weight: 60
+weight: 35
 alwaysopen: false
 categories: ["RC"]
-aliases: /rv/administration/system-logs/
+aliases: [ "/rv/administration/system-logs/",
+           "rc/administration/system-logs/",
+           "rc/administration/system-logs.md" ]
```
Comment (rrelledge) [inline content/rc/databases/system-logs.md:9]:
```suggestion
           "/rc/administration/system-logs/",
           "/rc/administration/system-logs.md" ]
```

**Review by rrelledge — CHANGES_REQUESTED** [review]
Looks good to me aside from the small alias change I left a suggestion for.

File: content/rc/databases/system-logs.md (hunk 1 above)
Comment (lanceleonard) [reply content/rc/databases/system-logs.md]: Fixed

**Review by rrelledge — APPROVED** [review]

**Pull request merged**